src/modules/tracker.js

```diff
@@ -133,16 +133,21 @@ class Tracker {
    * @private
    * @function trackInputFocusV2
    * @param {string} userInput - Input at the time user focused on the search bar
+   * @param {object} parameters - Additional parameters to be sent with request
+   * @param {object} [parameters.analyticsTags] - Pass additional analytics data
    * @param {object} [networkParameters] - Parameters relevant to the network request
    * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
    * @returns {(true|Error)}
    * @description User focused on search input element
    * @example
    * constructorio.tracker.trackInputFocusV2("text");
    */
-  trackInputFocusV2(userInput = '', networkParameters = {}) {
+  trackInputFocusV2(userInput = '', parameters = {}, networkParameters = {}) {
     const baseUrl = `${this.behavioralV2Url}focus?`;
     const bodyParams = {};
+    const {
+      analyticsTags,
+    } = parameters;
     let networkParametersNew = networkParameters;
     let userInputNew = userInput;
 
```

Comment:
We can get rid of the 
```
    if (typeof userInput === 'object') {
      networkParametersNew = userInput;
      userInputNew = '';
    }
```
Since this won't be swapped with the legacy implementation anymore.

Reply:
I'm not totally sure I understand what to change here but I've moved userInput into parameters, let me know if that's not what we want

Reply:
I was talking about removing this piece:
![image](https://github.com/Constructor-io/constructorio-client-javascript/assets/2954126/86f4de0a-055f-44db-8759-8fe2ab3f69e8)
(I see you already did that)

Reply:
Can we make `userInput` a parameter of its own? We follow that pattern is other places like `trackSearchSubmit(term, parameters, networkParameters = {})`.

Reply:
Got it, moved `userInput` back into parameters